ert3/config/_stages_config.py

```diff
@@ -43,16 +22,32 @@ class OutputRecord(_StagesConfig):
     location: str
 
 
+class TransportableCommand(_StagesConfig):
+    name: str
+    location: FilePath
+
+    @validator("location")
+    def is_executable(cls, location):
```

Comment:
Because if not we would need to get information about how to run it no? This is not a decision I'm saying that should stick to, it is more a reflection of the current requirements... I'm happy to have a discussion about how we want the unix step to be configured and what it represented, but my aim here was to get it up and running with the minimal effort. My suggestion is hence that we put this here now (given that we agree that currently it would be necessary) and then we iterate from there :)